### DISCUSSION TITLE
fzf-sesh

### DISCUSSION THREAD

**Discussion by PookieBuns** [joshmedeski/sesh#110] [Show and tell]
This script aims to bridge the gap and simplify integrating sesh with fzf
- One command to launch a sesh picker with fzf
- Supports fzf picker with icons and handles postprocessing of the selected session to connect to
- (Optional) replace `sesh` and `sesh connect` without arguments to launch the fzf sesh picker

Feel free to check it out!
https://github.com/PookieBuns/fzf-sesh

**Comment by joshmedeski** [comment]
In early development I actually worked to integrate fzf directly with the go binary. But I found allowing people to add a couple lines to their tmux config file offers a lot more flexibility and let's me focus on the core functionality of sesh.

**Reply by PookieBuns** [reply]
I agree. Not everyone uses fzf for their cli picker or uses a picker at all. My goal is to help those that do in fact use fzf as their primary CLI picker. Sort of like an optional add on.
